es6.md

```diff
@@ -75,9 +82,16 @@ A `yield` expression.
 ## ObjectPattern
 
 ```js
+interface PatternProperty <: Property {
```

Comment:
Any specific reason to use `PatternProperty` instead of `PropertyPattern` (which would be more consistent with other Pattern names)?